cfme/fixtures/appliance.py

```diff
@@ -194,14 +195,22 @@ def get_vddk_url(provider):
         major = str(provider.version)
         minor = "0"
     vddk_version = "v{}_{}".format(major, minor)
+
     try:
-        url = conf.cfme_data.basic_info.vddk_url.get(vddk_version)
+        vddk_urls = conf.cfme_data.basic_info.vddk_url
     except (KeyError, AttributeError):
-        pytest.skip('VDDK URL/Version not found in cfme_data.basic_info')
+        pytest.skip("VDDK URLs not found in cfme_data.basic_info")
+
+    if vddk_version not in vddk_urls:
+        logger.warning("Using VDDK {}, as VDDK {} was unavailable".formate("v6_5", vddk_version))
```

Comment:
typo, `format`

Reply:
ah, also, almost forgot. With logger, we want to pass it the strings to format in.

`logger.warning('my message with arg: %s', arg_to_format_in_to_log)`

Reply:
Thanks!